### DISCUSSION TITLE
Copy current time to clipboard?

### DISCUSSION THREAD

**Discussion by mandrael** [iina/iina#4602] [General]
Hi,
I use PotPlayer in Windows. It has the option to set a **hotkey for copying the current time** into the clipboard. I paste it in notepad and afterwards use it with losslesscut to split the video into the proper parts.
I search a player, which can do this on MacOS, but didn't succeed so far. Can IINA do it somehow or can it be implemented somehow?
Thanks, Michael

**Comment by low-batt** [comment]
Yes, it is possible to accomplish this with the current version of IINA using a small amount of scripting. See issue #3370 for a full discussion. This [comment](https://github.com/iina/iina/issues/3370#issuecomment-1163642686) shows how to configure [Automator](https://support.apple.com/guide/automator/welcome/mac) with a shortcut key that does this. See the earlier comments for details on installing [socat](https://formulae.brew.sh/formula/socat) and [jq](https://formulae.brew.sh/formula/jq).

Note that this will be easier to accomplish once IINA's new plugin system is available.

**Comment by TeslaLyon** [comment]
Have you found a quick and efficient way?

**Comment by low-batt** [comment]
What is it about the solution in issue #3370 that is not working in your case?
